src/app/commons/MarkdwonEditor.scss

```diff
@@ -93,8 +89,13 @@
 
   .active-markdown {
     display: block;
-    padding: 15px;
     overflow: scroll;
+    background: #FFFFFF;
+    border: 1px solid #E4E1E1;
+    box-sizing: border-box;
+    border-radius: 10px;
+    width: 756px;
```

Comment:
remember to extract common CSS !!
e.g., `background: #FFFFFF`, `display: block`, `box-sizing: border-box`, etc.